en/tutorials/linux_sbus.md

```diff
@@ -0,0 +1,43 @@
+# SBus Driver for Linux
```

Comment:
As previously commented (not by me) proper nouns (names) are capitalised. E.g. Futaba, not futaba, Linux. Similarly, protocols have proper spellings (I *think* SBus, not sbus)